### ISSUE TITLE
Images not displayed with no param

### ISSUE THREAD

**Issue by gregBerthelot** [venveo/serverless-sharp#106]
**Describe the bug**
I updated from 2.06 to latest and deploy. Now our images are not displayed behind CloudFront with no params, eg. xoxo.cloudfront.net/image.jpg?w=200 is well displayed but not xoxo.cloudfront.net/image.jpg for which I get `{}`

**Screenshots**
https://recordit.co/XaD4Qtdetv

**Comment by gregBerthelot** [comment]
@Mosnar any help would be appreciated on this. We are stuck with 2.06 and node10 and have the need to update

**Comment by gregBerthelot** [comment]
The above reported issue is from Node14 and serverless-sharp 2.1.0

**Comment by gregBerthelot** [comment]
Seems that almost one params is needed even if it is not a valid one. 

Please check 
https://di26vpqopmq9u.cloudfront.net/daylight-forest-glossy-lake-443446.jpg?foo 
which is well displaying the image while 
https://di26vpqopmq9u.cloudfront.net/daylight-forest-glossy-lake-443446.jpg 
does not

**Comment by Mosnar** [comment]
Yeah, I noticed that while working on version 3 and fixed it. I didn't realize it existed prior to that. I'll see if I can cut a release. I'm trying to focus on getting the next version out the door right now, hence why I haven't spent much time on issues lately.

**Comment by gregBerthelot** [comment]
@Mosnar AWS are ending support for Node.js 12 in AWS Lambda. This follows Node.js 12 End-Of-Life (EOL) reached on April 30, 2022. Any chance you fix the above issue so we can upgrade to Node.js 14?
